exercise/templatetags/exercise.py

```diff
@@ -213,6 +213,18 @@ def min_group_size(context):
 
 @register.simple_tag(takes_context=True)
 def module_accessible(context, entry):
+    t = entry.get('reading_opening_time', entry.get('opening_time'))
```

Comment:
Your new function `exercise_accessible` seems to be almost identical to the old `module_accessible`? What is the difference and is it really necessary to duplicate code? What if you add new parameters to the old function?

Reply:
The difference is whether the function uses the `reading_opening_time` or the `opening_time`. Those functions have now their common parts in the `is_accessible`.